docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+    zero spin throttle has been configured differently. For more information on how the module's response to throttle commands will change, see the :ref:`manual_zero_spin` section.
+    These descriptions also assume that the module has already :ref:`armed <manual_advanced_arming>` before spinning.
+
+.. _throttle_mode:
+
+Mode
+*****
+Every Vertiq module should be configured to operate in a certain “mode”. This mode determines how the module will interpret throttle commands it receives and 
+how it will attempt to control itself. There are three possible modes, covered in the sections below.
+
+The mode of the module can be controlled using the *Mode* parameter in the General tab of the IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/throttle/throttle_mode.png
+    :align: center
+    :width: 60%
+    :alt: Mode Parameter
+
+    Mode Parameter in IQ Control Center
+
+PWM
+####
+Throttle commands will be interpreted as commanding a percentage of the module’s supply voltage in this mode. E.g. a 50% throttle command will command the module to 
+apply 50% of its supply voltage as its drive voltage. If the module were powered with a 24V supply, that 50% command would result in a drive voltage of 12V. 
+
+Note that if using a battery, as the battery drains the meaning of commands in this mode will change with it. For example, if the supply voltage in the previous example 
+dipped to 23V, a 50% command would now result in a drive voltage of 11.5V.
+
+.. _throttle_voltage_mode:
+
+Voltage
+########
+Throttle commands will be interpreted as commanding a percentage of a user configurable voltage range in this mode. This range is set by the *Max Volts* parameter, 
+as covered in the :ref:`throttle_maximums` section below. For example, if the maximum voltage is set to 20V and a 50% throttle command is received, a drive voltage of 10V will 
+be applied, because that is 50% of the 0V to 20V range.
+
+Note that in this mode the drive voltage a module attempts to apply is independent of the supply voltage of the module, unlike in the PWM mode. Changing battery 
+voltage will have no effect on the drive voltages applied in Voltage mode.
+
+.. _throttle_velocity_mode:
+
+Velocity
+#########
+Throttle commands will be interpreted as commanding a percentage of a user configurable velocity range in this mode. This range is set by the Max Velocity parameter, 
+as covered in the :ref:`throttle_maximums` section below. In this mode the module will operate as a closed loop controller, applying the drive voltage needed to maintain the 
+commanded velocity. For example, if the maximum velocity is set to 200 rad/s and a 50% throttle command is received, the module will set the drive voltage to attempt 
+to maintain a velocity of 100 rad/s, since that is 50% of the 0 rad/s to 200 rad/s range.
+
+.. _throttle_maximums:
+
+Maximums

```

Comment:
since velo and voltage both use the maximums, why not list them first

Reply:
List the maximums first? List velocity and voltage first in the Modes? 

I do prefer having the Modes listed first, since I think that helps clarify the meaning of the maximums better.

Reply:
never mind on this one. what's there is fine